spec/integration_spec.rb

```diff
@@ -514,6 +514,18 @@
     Book.index(safe_index_uid('Book')).delete_all_documents
   end
 
+  it 'returns array of tasks on #ms_index!' do
+    moby_dick = Book.create! name: 'Moby Dick', author: 'Herman Melville', premium: false, released: true
+
+    tasks = moby_dick.ms_index!
+
+    expect(tasks).to contain_exactly(
+      a_hash_including('uid'),
+      a_hash_including('taskUid'),
+      a_hash_including('taskUid')
```

Comment:
Did you forget to add a different attribute here? In any case remove one of the `taskUid`